lib/smart-contract-client.js

```diff
@@ -37,37 +39,31 @@ const contractABI = [
   }
 ]
 
-// Contract address on the Filecoin EVM
-const CONTRACT_ADDRESS = '0x14183aD016Ddc83D638425D6328009aa390339Ce'
-// Singleton instance of the smart contract client
-let smartContract
 // Initialize provider and contract
 async function getSmartContractClient (rpcUrl) {
```

Comment:
Since this is creating a new instance of the smart contract client, I think it's more appropriate to use the verb `create`. The verb `get` usually implies obtaining an existing instance.

Also, this function is synchronous, so let's remove the `async` keyword in the function definition and remove `await` in places calling ig.

```suggestion
function createSmartContractClient (rpcUrl) {
```

Reply:
Addresse in https://github.com/CheckerNetwork/spark-checker/commit/5b63a20e6274e9440e5b7dba77d7bc0b4d1d7b54